### DISCUSSION TITLE
Update the Facebook SDK for IOS

### DISCUSSION THREAD

**Discussion by GonzaloAszyn** [thebergamo/react-native-fbsdk-next#401] [Q&A] [answered]
Hello there! I'm utilizing this repository to record app events in my React Native project. However, my Events Manager dashboard is displaying a message stating that my app needs to be updated to run on iOS 14, and that I need to update to the latest Facebook SDK for iOS. I am currently using the most recent release of react-native-fbsdk-next, which uses version 15 of the Facebook SDK. I'm wondering if there's anything I can do to resolve this issue. Have there been any updates or news regarding a release that will address this compatibility problem? I would greatly appreciate any assistance or insights you may have. Thank you!

**Comment by mikehardy** [comment, marked as answer]
Blocked pending this https://github.com/thebergamo/react-native-fbsdk-next/pull/339 unfortunately
Any help unblocking it would be appreciated, this repository is a community effort and though I've been the prime mover here in the past I am not *currently* needing the upgrade here so it is not a priority for me. If it is a priority for you (or anyone else reading) please feel free to pick up the PR and drive it to conclusion

**Reply by GonzaloAszyn** [reply]
That's okay! Thanks for the reply is such short notice. Unfortunately I don't have the time to collaborate with this repository even though I would love to. Once again thanks for your efforts!

**Comment by joaolvcas** [comment]
How can I set the GraphAPI version? Looks like I'm using a version older than 11.0.
